pytest/integration/test_parameters_integration.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Integration tests to make sure hbar values set correctly in returned results"""
+import inspect
+import itertools
+import logging
+
+import pytest
+
+import numpy as np
+import tensorflow as tf
+
+import strawberryfields as sf
+from strawberryfields import ops
+from strawberryfields.parameters import Parameter
+
+
+@pytest.mark.backends("fock", "tf")
+def test_parameters_with_operations(batch_size, setup_eng):
```

Comment:
```suggestion
def _test_parameters_with_operations(batch_size, setup_eng):
```

Reply:
If it is confusing, and apparently not doing anything, let's formally pull it out of the test suite for now

Reply:
I would suggest leaving it, but marking it to be fixed in a later PR